.github/travis/setup.sh

```diff
@@ -1,42 +0,0 @@
-#!/bin/bash
```

Comment:
I think it would still be useful to output the exact version information for the toolchain if we don't do it elsewhere?

Reply:
CMake outputs the versions of `$CC` and `$CXX` ([example](https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/1760/checks?check_run_id=2731164444#step:6:15)), do we need anything else? Note that the commands in this script never ran on GHA because `$TRAVIS_BUILD_STAGE_NAME` is empty.

Reply:
SGTM!